src/pages/Login/Login.js

```diff
@@ -1,9 +1,85 @@
 import React from 'react';
 import './Login.scss';
+import { Link } from 'react-router-dom';
 
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      idInput: '',
+      pwInput: '',
+    };
+  }
+
+  goToMain = () => {
+    this.props.history.push('/Main');
+  };
+
+  updateIdInput = e => {
+    this.setState({
+      idInput: `${e.target.value}`,
+    });
+  };
+
+  updatePwInput = e => {
+    this.setState({
+      pwInput: `${e.target.value}`,
+    });
+  };
```

Comment:
```suggestion
  updateInputs = e => {
  const { name, value } = e.target;
    this.setState({
      [name]: value,
    });
  };
```
중복되는 함수 부분을 하나로 만드는 연습을 해보세요!
해당 부분 이해해보시고 어렵다면 질문주세요!!
단 name 키워드에 접근하기 위해 input 태그에 name 값을 부여해주셔야 합니다!
태그 부분에도 리뷰 남길게요. 확인해주세요~